offer-manager-app/src/main/groovy/life/qbic/portal/offermanager/components/offer/create/CreateOfferPresenter.groovy

```diff
@@ -34,11 +34,17 @@ class CreateOfferPresenter implements CreateOfferOutput, FetchOfferOutput{
         this.createOfferViewModel.setOfferCreatedSuccessfully(true)
     }
 
+    /**
+     * @inheritDocs
+     */
```

Comment:
This should not be necessary and is default behaviour for overwritten methods.
> Inherit from classes and interfaces - Inheriting of comments occurs in all three possible cases of inheritance from classes and interfaces:
When a method in a class overrides a method in a superclass
When a method in an interface overrides a method in a superinterface
When a method in a class implements a method in an interface

https://docs.oracle.com/javase/7/docs/technotes/tools/solaris/javadoc.html

Reply:
You also need to wrap the `@inheritDocs` in enclosing `{}` for it to work

Reply:
the IDE was able to display the JavaDocs, nice catch!